src/commands/mod.rs

```diff
@@ -0,0 +1,15 @@
+#[cfg(feature = "init")]
+pub mod init;
+
+pub mod build;
+pub mod local;
+pub mod template;
+
+pub trait BlueBuildCommand {
+    fn run(&mut self);
```

Comment:
I'm thinking, maybe create a default implementation that checks if try_run errors, prints the error, and exits the process. That'll help reduce more repeated code

Reply:
oh yeah this will help alot